djangosaml2/views.py

```diff
@@ -76,6 +76,21 @@
     validate_referral_url,
 )
 
+# Update Content-Security-Policy headers for POST-Bindings
+try:
+    from csp.decorators import csp_update
+except ModuleNotFoundError:
+    # If csp is not installed, do not update fields as Content-Security-Policy
+    # is not used
```

Comment:
please add  a `logger.warning` here. It would be appreciated to make aware the deployers that they may haev security risks (if CSP is not configured at least in the httpd)

Reply:
I polished up the security page and added a warning, please check if I am missing important information :)